tests/unit/plugins/modules/test_xdg_mime.yaml

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2025, Marcos Alano <marcoshalano@gmail.com>
+# Based on gio_mime module. Copyright (c) 2022, Alexei Znamensky <russoz@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# TODO: add tests for setting multiple mime types at once
+---
+anchors:
+  environ: &env-def {environ_update: {LANGUAGE: C, LC_ALL: C}, check_rc: true}
+test_cases:
+  - id: test_set_handler
+    input:
+      handler: google-chrome.desktop
+      mime_type: x-scheme-handler/http
+    output:
+      current_handlers: ['']
+      changed: true
+    mocks:
+      run_command:
+        - command: [/testbin/xdg-mime, --version]
+          environ: *env-def
+          rc: 0
+          out: "xdg-mime 1.2.1\n"
```

Comment:
Since this is repeated throughout the file, you might want to make an anchor out of it at the top and just replicate, like we do with the environment settings.

Reply:
I used anchors in some parts of the yaml file.